.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Hi @radeksimko - thank you for adding this! What do you think about changing the interval to monthly? I'm concerned that daily could be a little intrusive.

```suggestion
      interval: "monthly"
```